### PR TITLE
Adds storage repositories

### PR DESCRIPTION
**Pull request by dguitarbite**

### CODE REVIEW

**Comment by tboerger** [comment]
LGTM

File: releases/development/master/extra/install-chef-suse.sh [hunk 1]
```diff
@@ -582,6 +584,9 @@ check_repo_product SLE11-HAE-SP3-Pool     'SUSE Linux Enterprise High Availabili
 check_repo_product SLE11-HAE-SP3-Updates  'SUSE Linux Enterprise High Availability Extension 11 SP3' 'false'
 check_repo_product SUSE-Cloud-5-Pool    'SUSE Cloud 5'
 check_repo_product SUSE-Cloud-5-Updates 'SUSE Cloud 5'
+check_repo_product SUSE-Storage-1.0-Pool 'SUSE Enterprise Storage'
+check_repo_product SUSE-Storage-1.o-Updates 'SUSE Enterprise Storage'
```
Comment (vuntz) [inline releases/development/master/extra/install-chef-suse.sh]: The check needs to be optional (like the HAE check earlier)

**Comment by dirkmueller** [comment]
-1, you also need to add it to SKIP_CHECKS (see line 524)

File: releases/development/master/extra/install-chef-suse.sh (hunk 1 above)
Comment (dguitarbite) [inline releases/development/master/extra/install-chef-suse.sh]: I see the line 529 should make storage optional, am I missing something here?

Comment (dirkmueller) [inline releases/development/master/extra/install-chef-suse.sh]: you're missing the 'false'

File: releases/development/master/extra/install-chef-suse.sh [hunk 2]
```diff
@@ -582,6 +587,9 @@ check_repo_product SLE11-HAE-SP3-Pool     'SUSE Linux Enterprise High Availabili
 check_repo_product SLE11-HAE-SP3-Updates  'SUSE Linux Enterprise High Availability Extension 11 SP3' 'false'
 check_repo_product SUSE-Cloud-5-Pool    'SUSE Cloud 5'
 check_repo_product SUSE-Cloud-5-Updates 'SUSE Cloud 5'
+check_repo_product SUSE-Storage-1.0-Pool 'SUSE Enterprise Storage' 'false'
```
Comment (dirkmueller) [inline releases/development/master/extra/install-chef-suse.sh]: it is called
'SUSE Enterprise Storage 1.0'

File: releases/development/master/extra/install-chef-suse.sh [hunk 3]
```diff
@@ -357,7 +357,9 @@ if [ -n "$PROVISIONER_JSON" ]; then
               SLE12-Cloud-Compute \
               SLE12-Cloud-Compute-PTF \
               SLE-12-Cloud-Compute5-Pool \
-              SLE-12-Cloud-Compute5-Updates
+              SLE-12-Cloud-Compute5-Updates \
+              SUSE-Storage-1.0-Pool \
+              SUSE-Storage-1.o-Updates
```
Comment (dirkmueller) [inline releases/development/master/extra/install-chef-suse.sh]: 1.0, not 1.o

File: releases/development/master/extra/install-chef-suse.sh [hunk 4]
```diff
@@ -582,6 +587,9 @@ check_repo_product SLE11-HAE-SP3-Pool     'SUSE Linux Enterprise High Availabili
 check_repo_product SLE11-HAE-SP3-Updates  'SUSE Linux Enterprise High Availability Extension 11 SP3' 'false'
 check_repo_product SUSE-Cloud-5-Pool    'SUSE Cloud 5'
 check_repo_product SUSE-Cloud-5-Updates 'SUSE Cloud 5'
+check_repo_product SUSE-Storage-1.0-Pool 'SUSE Enterprise Storage' 'false'
+check_repo_product SUSE-Storage-1.o-Updates 'SUSE Enterprise Storage' 'false'
```
Comment (dirkmueller) [inline releases/development/master/extra/install-chef-suse.sh]: 1.0, not 1.o

File: releases/development/master/extra/install-chef-suse.sh [hunk 5]
```diff
@@ -523,6 +525,9 @@ REPOS_SKIP_CHECKS+=" Cloud SLES11-SP3-Updates SUSE-Cloud-5-Pool SUSE-Cloud-5-Upd
 # HAE add-on should remain optional for now
 REPOS_SKIP_CHECKS+=" SLE11-HAE-SP3-Pool SLE11-HAE-SP3-Updates"
 
+# Storage should remaim optional for now
+REPOS_SKIP_CHECKS+=" SUSE-Storage-1.0-Pool SUSE-Storage-1.o-Updates"
```
Comment (tboerger) [inline releases/development/master/extra/install-chef-suse.sh]: 1.0, not 1.o

**Comment by dirkmueller** [comment]
+1

File: releases/development/master/extra/install-chef-suse.sh [hunk 6]
```diff
@@ -523,6 +525,9 @@ REPOS_SKIP_CHECKS+=" Cloud SLES11-SP3-Updates SUSE-Cloud-5-Pool SUSE-Cloud-5-Upd
 # HAE add-on should remain optional for now
 REPOS_SKIP_CHECKS+=" SLE11-HAE-SP3-Pool SLE11-HAE-SP3-Updates"
 
+# Storage should remaim optional for now
```
Comment (vuntz) [inline releases/development/master/extra/install-chef-suse.sh]: Typo: remain

Comment (dguitarbite) [inline releases/development/master/extra/install-chef-suse.sh]: Thanks for catching it.

**Pull request merged**